app/controllers/eligibility_interface/base_controller.rb

```diff
@@ -3,6 +3,12 @@ class BaseController < ApplicationController
     before_action :load_region
     after_action :save_eligibility_check_id
 
+    http_basic_authenticate_with name: ENV.fetch("TEST_USERNAME", "test"),
+                                 password: ENV.fetch("TEST_PASSWORD", "test"),
```

Comment:
```suggestion
    http_basic_authenticate_with name: ENV.fetch("TEST_USERNAME", ENV.fetch("SUPPORT_USERNAME", "test")),
                                 password: ENV.fetch("TEST_PASSWORD", ENV.fetch("SUPPORT_PASSWORD", "test")),
```

I'm thinking this might be better to do this so we only need to set the `TEST_USERNAME` and `TEST_PASSWORD` in the `test` environment and can leave the rest with the existing `SUPPORT_` environment variables.